### ISSUE TITLE
How to configure Digital Twin and Connector ?

### ISSUE THREAD

**Issue by HenkySerpent** [eclipse-tractusx/sldt-digital-twin-registry#228]
Could not able to find any information regarding this configuration. Consider the example provided in the doc 
there are some api mentioned in ASSET creation in Connector of provider "baseUrl":{BACKEND_SERVICE} actually it should be URL to digital twin right ?

Where can I find more information regarding creation/configuration of these API endpoints in data transfer example of /docs/README.md

* The EDC compatible endpoint where the semanticId compatible data can be fetched. Catena-X defined the URL structure as: Read Endpoint: http:////<AAS_ID>-<SUBMODEL_ID>/submodel?content=value Operations Endpoint: <SAME_AS_READ//invoke?content=value

* https://edc.data.plane/mypath/submodel - in SHELL_CREATION payload. How to configure this URL in connector? this look like a dynamic URL , is it?

Basically please provide information the SAMPLE URLS in the payloads of sample data transfer and please give information on the confguration needed in provider connector and DTR so that the data transfer occurs

@tunacicek Please provide some insights on this.

**Comment by tunacicek** [comment]
Hi @HenkySerpent ,
yes the baseUrl (Backend_Service) to create an edc-Asset should point to the digital Twin Registry, which is behind the "provider"-Edc.

More information can be find here : https://eclipse-tractusx.github.io/docs-kits/category/digital-twin-kit

It is up to the provider on how to create the edc-Asset for the submodels. Important is at least the correct shell creation which points to the (submodel)- edc-Asset
